internal/resources/oncall/data_source_integrations.go

```diff
@@ -0,0 +1,49 @@
+package oncall
+
+import (
+	"context"
+
+	onCallAPI "github.com/grafana/amixr-api-go-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceIntegration() *schema.Resource {
+	return &schema.Resource{
+		Description: `
+* [HTTP API](https://grafana.com/docs/oncall/latest/oncall-api-reference/integrations/)
+`,
+		ReadContext: withClient[schema.ReadContextFunc](dataSourceIntegrationRead),
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The integration ID.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
```

Comment:
It'd be nice if that could be specified as well. Input just the name -> get the ID and vice-versa

Reply:
The API https://grafana.com/docs/oncall/latest/oncall-api-reference/integrations/#get-integration just accepts an ID so I don't know how we would go about with that, would need advice if that would be a strict requirement